cmd/registry/cmd/label.go

```diff
@@ -29,157 +29,67 @@ import (
 	"google.golang.org/genproto/protobuf/field_mask"
 )
 
-var labelFilter string
-var labelOverwrite bool
-var labelsToSet map[string]string
-var labelsToClear []string
+var labelKeyFilter string
+var labelKeyOverwrite bool
+var labelKeysToSet map[string]string
+var labelKeysToClear []string
```

Comment:
For the same reason as above, consider declaring these in the command's run function.

Reply:
done